<launch>

   <arg name="with_camera" default="false"/>
   <arg name="with_viewer" default="false"/>
   
   <arg name="input_image_topic" default="image_raw"/>
   <arg name="img_haz_topic"     default="img_hazmats"/>

   
   <node name="hazmat_detection" pkg="find_object_2d" type="find_object_2d">
    <remap from="/image"              to="$(arg input_image_topic)"/>
    <param name="gui"                 value="$(arg with_viewer)"/>
    <param name="subscribe_depth"     value="false"/>
    <param name="objects_path"        value="$(find ohm_rrl_perception_utility)/config/hazmats"/>               <!-- Path to folder containing objects to detect. -->
    <param name="settings_path"       value="$(find ohm_rrl_perception_utility)/config/hazmats/settings.ini"/>  <!-- Path to settings file (*.ini). -->
   </node>
   
   <node name="hazmat_viz" pkg="ohm_rrl_perception_utility" type="hazmat_viz_node" output="screen">
    <param name="input_topic" value="$(arg input_image_topic)"/>
    <param name="viz_topic"   value="$(arg img_haz_topic)"/>
   </node>
   
</launch>